jwst/ami/tests/test_ami_interface.py

```diff
@@ -1,39 +1,14 @@
-import numpy as np
 import pytest
 
 import stpipe
-from stpipe import crds_client
 
 from stdatamodels.jwst import datamodels
 
 from jwst.ami import AmiAnalyzeStep
 
 
 @pytest.fixture()
-def mock_nrm_reference_file(tmp_path, monkeypatch):
-    """
-    At the moment nrm only exists on CRDS-TEST so this
-    fixture will mock fetching any 'nrm' reference file
-    """
-    fn = tmp_path / "nrm.fits"
-
-    # make a fake nrm file
-    m = datamodels.NRMModel()
-    m.nrm = np.zeros((1024, 1024), dtype='f4')
-    m.save(fn)
-
-    original_get_reference_file = crds_client.get_reference_file
-
-    def mock_get_reference_file(dataset, reference_file_type, observatory):
-        if reference_file_type == 'nrm':
-            return str(fn)
-        return original_get_reference_file(dataset, reference_file_type, observatory)
-
-    monkeypatch.setattr(crds_client, "get_reference_file", mock_get_reference_file)
```

Comment:
I see that this was never used in the original code for `example_model()`, but the monkeypatch makes me wonder what this was doing, if anything.

Was there some way in which `example_model()` used to require this fixture to exist but now no longer does require it?
Or, was it simply never used?

Reply:
This existed to mock the reference file before CRDS had any valid matches.  It's no longer needed, now that there are NRM files in CRDS.